templates/taxi/driver_list.html

```diff
@@ -9,6 +9,9 @@ <h1>Driver list</h1>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
           ({{ driver.first_name }} {{ driver.last_name }})
         </li>
+        {% if driver.username == user.username %}
```

Comment:
The condition `{% if driver.username == user.username %}` is correctly used to identify the logged-in user in the list. Ensure that the `user` context variable is available in the template.